### PR TITLE
My first Pull Request

### PR DESCRIPTION
**Pull request by Wolfmikerson**
This pull request includes:

- Added index.html
- Added style.css
- Configured linters for errors check
- Added .gitignore to ignore files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,45 @@
...
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```
Comment (Emmyn5600) [inline README.md]: - [ ] Please refer to this t[emplate](https://github.com/microverseinc/readme-template) to organize and modify your text where is necessary in the Readme file

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+# .gitignore
+node_modules/
+
+# ignore test.md
+test.md
```
Comment (Emmyn5600) [inline .gitignore:6]: Great work here, you included the both files.

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
```
Comment (Emmyn5600) [inline .github/workflows/linters.yml:48]: Nice one here

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Wolfmikerson   

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  
 * #### PR describes the short summary of what you did or implemented in the project using the past tense. For example, `Added index.html file` `Configured linters to check for errors`.
 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by AshakaE — APPROVED** [review]
### Hi @Wolfmikerson  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**